packages/components-sketch/views/link.hbs

```diff
@@ -65,47 +65,87 @@ file, You can obtain one at https://mozilla.org/MPL/2.0/.
     <div class="agent-states--grid" data-category="Not Visited">
         <div class="agent-states--item">
             <p class="agent-states--label">:enabled</p>
-            <scale-link data-sketch-key="link-10" icon="{{>icon}}" href="#">Link Text</scale-link>
+            <scale-link data-sketch-key="link-10" data-has-icon="true" href="#">Link Text <scale-icon-navigation-external-link
+        size="16"
+        accessibility-title="open link in new tab"
+        slot="icon"
+    ></scale-icon-navigation-external-link></scale-link>
         </div>
         <div class="agent-states--item">
             <p class="agent-states--label">:hover</p>
-            <scale-link data-sketch-key="link-11" icon="{{>icon}}" href="#" data-sketch-state="hover">Link Text</scale-link>
+            <scale-link data-sketch-key="link-11" data-has-icon="true" href="#" data-sketch-state="hover">Link Text <scale-icon-navigation-external-link
+        size="16"
+        accessibility-title="open link in new tab"
+        slot="icon"
+    ></scale-icon-navigation-external-link></scale-link>
         </div>
         <div class="agent-states--item">
             <p class="agent-states--label">:focus</p>
-            <scale-link data-sketch-key="link-12" icon="{{>icon}}" href="#" data-sketch-state="focus">Link Text</scale-link>
+            <scale-link data-sketch-key="link-12" data-has-icon="true" href="#" data-sketch-state="focus">Link Text <scale-icon-navigation-external-link
+        size="16"
```

Comment:
unfortunately, prettier has no support for handlebars but you could run the html parts through to tidy up the formatting of this file - the indentation is really off here 😅

Reply:
did end up settings the handebars file associaction and applied html formatting

> sidenote: we should move on from hbs templates, i have something leaner in mind and will explore replacement soon 🙃

Reply:
please test if everything still works after formatting - prettier might do weird things to the hbs code.